doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -370,6 +370,8 @@ Matter samples
 Networking samples
 ------------------
 
+* Updated the :kconfig:option:`CONFIG_HEAP_MEM_POOL_SIZE` Kconfig option value to at least 1280 for all networking samples.
+  This is a requirement from zephyr and removes a build warning.
 * Updated the following samples to use the :ref:`lib_downloader` library instead of the :ref:`lib_download_client` library:
```

Comment:
Suggestion for line no: 373 to 379
```suggestion
* Updated:
  
  * The :kconfig:option:`CONFIG_HEAP_MEM_POOL_SIZE` Kconfig option value to ``1280`` for all networking samples.
    This is a requirement from Zephyr and removes a build warning.
  * The following samples to use the :ref:`lib_downloader` library instead of the :ref:`lib_download_client` library:

    * :ref:`aws_iot`
    * :ref:`azure_iot_hub`
    * :ref:`download_sample`
```

Reply:
Updated, though it is not for all networking samples, only those that had a lower value than 1280.